zap/src/output/tooling.rs

```diff
@@ -12,16 +12,43 @@ struct ToolingOutput<'src> {
 	tabs: u32,
 	buf: String,
 	var_occurrences: HashMap<String, usize>,
+	client_evpaths: HashMap<(usize, bool), String>,
+	server_evpaths: HashMap<(usize, bool), String>,
+	client_fnpaths: HashMap<usize, String>,
+	server_fnpaths: HashMap<usize, String>,
```

Comment:
This feels like... a lot of boilerplate. Why don't we just store the path of evdecls/fndecls like we do for tydecls?